src/main/java/com/getindata/connectors/http/internal/table/lookup/JavaNetHttpPollingClient.java

```diff
@@ -89,7 +90,14 @@ private Optional<RowData> processHttpResponse(
             HttpResponse<String> response,
             HttpLookupSourceRequestEntry request) throws IOException {
 
-        this.httpPostRequestCallback.call(response, request, "endpoint", Collections.emptyMap());
+        try {
+            this.httpPostRequestCallback.call(
+                    response, request, "endpoint", Collections.emptyMap()
+            );
+        } catch (PostRequestCallbackException e) {
+            log.debug("Error during post request callback.", e);
+            return Optional.empty();
```

Comment:
why do we not throw an Exception here? There is already precedence as this method already throws throws IOException